assets/src/edit-story/components/library/textLibrary.js

```diff
@@ -14,52 +14,93 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+
 /**
- * External dependencies
+ * Internal dependencies
  */
-import PropTypes from 'prop-types';
+import { Section, MainButton, Title, SearchInput, Header } from './common';
+import { FontPreview } from './text';
+
+const PRESETS = [
+  {
+    id: 'heading',
+    title: 'Heading',
+    fontSize: 28,
```

Comment:
These numbers should probably be higher now that the base story size used for calculation is 1080x1920

Reply:
Still doesn't look good with higher numbers, do we have some sort of mapping function? I don't want to inflate the numbers that the user sees

Reply:
We have `UnitsProvider` / `useUnits` and `dataToEditorY` / `dataToEditorX`. There's some prior art for using `dataToEditorY` for font sizes in the code base in `SingleSelectionMovable` and `MultiSelectionMovable`.

Reply:
`UnitsProvider` is only available at the canvas level (the text library is above that) and also I think `dataToEditorY` is already applied when the text is displayed if I understand correctly

Reply:
Ah, you're right!

Perhaps @dvoytenko knows more then? But I think we should actually be good with the current numbers. 

However, I just realized two things:

* The font sizes should be 28/22/16 [according to Figma](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=3462%3A2868)
* The line-heights  should be 36/28/24 [according to Figma](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=3462%3A2868)
* We need to disable fit-text for these presets; otherwise the font-size gets overridden automatically when resizing. Asked on Figma for clarification, but sounds like we need to open an issue for this.

Reply:
`UnitsProvider` can be defined on any level that makes sense, but I don't think it makes sense here. But we can always use utilities from `dimensions.js` if they help. Here we are only dealing with "data units" and `UnitsProvider` is mainly for rendering on some sort of canvas to convert "data units" for display/preview/output. I think, however, Figma uses 412:732 base, while our current data base is 1080:1920. I recommend we do a quick math between these two bases like [here](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/library/libraryContent.js#L57). If you'd like, you can move these calculation to utils, e.g. as `defaultToDataX/Y`.

Reply:
Added the utilities